aclk/aclk_query.c

```diff
@@ -319,24 +319,17 @@ static char *aclk_encode_response(char *src, size_t content_size, int keep_newli
 #pragma region ACLK_QUERY
 #endif
 
-static usec_t aclk_web_api_request_v1(RRDHOST *host, struct web_client *w, char *url)
+static usec_t aclk_web_api_request_v1(RRDHOST *host, struct web_client *w, char *url, usec_t q_created)
 {
-    usec_t t;
+    usec_t t_db = now_monotonic_high_precision_usec();
```

Comment:
I think its better to move this just before the `web_client_api_request_v1` call, to measure just the request call and not any delay that can happen within the `aclk_metric_mat_update`

Reply:
resolved in latest comit